specs/predeploys.md

```diff
@@ -93,290 +118,176 @@ In the Bedrock system, this contract will no longer be used as part of the
-    event Mint(address indexed _account, uint256 _amount);
-    event Burn(address indexed _account, uint256 _amount);
-    event Transfer(address indexed from, address indexed to, uint256 value);
-    event Approval(address indexed owner, address indexed spender, uint256 value);
-
-    function supportsInterface(bytes4 interfaceId) external view returns (bool);
-    function l1Token() external returns (address);
-    function mint(address _to, uint256 _amount) external;
-    function burn(address _from, uint256 _amount) external;
-    function totalSupply() external view returns (uint256);
-    function balanceOf(address account) external view returns (uint256);
-    function transfer(address to, uint256 amount) external returns (bool);
-    function allowance(address owner, address spender) external view returns (uint256);
-    function approve(address spender, uint256 amount) external returns (bool);
-    function transferFrom(
-        address from,
-        address to,
-        uint256 amount
-    ) external returns (bool);
-}
-```
+Address: `0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000`
 
-## WETH9
+The `LegacyERC20ETH` predeploy represents all ether in the system before the
+Bedrock upgrade. All ETH was represented as an ERC20 token and users could opt
+into the ERC20 interface or the native ETH interface.
 
-`WETH9` is the standard implementation of Wrapped Ether on Optimism.
+The upgrade to Bedrock migrates all ether out of this contract and moves it to
+its native representation. All of the stateful methods in this contract will
+revert after the Bedrock upgrade.
 
-```solidity
-interface WETH9 {
-    function name() public returns (string);
-    function symbol() public returns (string);
-    function decimals public returns (uint8);
+This contract is deprecated and its usage should be avoided.
 
-    event  Approval(address indexed src, address indexed guy, uint wad);
-    event  Transfer(address indexed src, address indexed dst, uint wad);
-    event  Deposit(address indexed dst, uint wad);
-    event  Withdrawal(address indexed src, uint wad);
+## WETH9
 
-    function balanceOf(address) public returns (uint);
-    function allowance(address, address) public returns (uint);
+[Implementation](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/vendor/WETH9.sol)
 
-    function deposit() public;
-    function withdraw(uint wad) public;
-    function totalSupply() public view returns (uint);
-    function approve(address guy, uint wad) public returns (bool);
-    function transfer(address dst, uint wad) public returns (bool);
+Address: `0x4200000000000000000000000000000000000006`
 
-    function transferFrom(
-        address src,
-        address dst,
-        uint wad
-    ) public returns (bool);
-}
-```
+`WETH9` is the standard implementation of Wrapped Ether on Optimism. It is a
+commonly used contract and is placed as a predeploy so that it is at a
+deterministic address across chains.
```

Comment:
I'm pretty sure it's not at `0x4200000000000000000000000000000000000006` on other chains though?

Reply:
I suppose i should clarify `op-chains` - getting at it being deployed there on any optimism network